### PR TITLE
Fix compilation of tests

### PR DESCRIPTION
**Pull request by uphoffc**
@Thomas-Ulrich Could you check the line with abs? I guess you wanted to use fabs there?

### CODE REVIEW

File: tests/f_120_sumatra.cpp [hunk 1]
```diff
@@ -59,7 +58,7 @@ void model(double x, double y, double z, double params[6]) {
   double bii[6];
   STRESS_STR_DIP_SLIP_AM(0.25, 0.3, strike, 8.0, 555562000.0, 0.4e6, 0.7, 1.0, 0.5, bii);
   
-  easi::STRESS_STR_DIP_SLIP_AM test;
+  /*easi::STRESS_STR_DIP_SLIP_AM test;
```
Comment (krenzland) [inline tests/f_120_sumatra.cpp:61]: Is there a reason why this is commented out and not deleted?

**Review by krenzland — CHANGES_REQUESTED** [review]
Just one small thing (commented out code). Otherwise looks fine. Have not tested it (yet).

By the way, you can also use std::abs for floating point values.

**Comment by Thomas-Ulrich** [comment]
yes fabs

**Review by Thomas-Ulrich — APPROVED** [review]
LGTM

File: tests/f_120_sumatra.cpp (hunk 1 above)
Comment (uphoffc) [reply tests/f_120_sumatra.cpp:61]: It's a reminder to put it back in :D
The problem here is the header-only approach of easi, which is something I'd like to get rid of.

Comment (krenzland) [reply tests/f_120_sumatra.cpp:61]: Hm, in general I'd argue that this is what git history is for. Could we compromise and add a comment why this is commented out?

Why is the header-only approach a problem? If you want to switch to a non-header-only approach, we should maybe talk and discuss how we can improve the build system to accommodate that. Just as a prototype how we can build libraries in the future in a sustainable way with cmake.

Comment (uphoffc) [reply tests/f_120_sumatra.cpp:61]: Yes we can add a comment. I might also work on switching to a regular library soon.

The problem is that your stuff doesn't compile if you include easi headers in multiple files. (For that to work every function needs to be static or inline and that really increases compile time.)

There are workarounds, e.g. as in https://github.com/nothings/stb/blob/master/stb_image.h . But then you still have the problem that you might use easi in two libs and you can't link them together....

So initially I thought this library will be more template heavy. Then header-only makes sense as you anyway need the full definition of every function and you don't run into the linking problem. In the current state, a "normal" library would be much better.

Comment (krenzland) [reply tests/f_120_sumatra.cpp:63]: Well, there's also the upside of header-only that it makes linking trivial but if we set it up properly, it shouldn't be an issue.

**Review by krenzland — APPROVED** [review]

**Pull request merged**